robot_ws/src/voice_interaction_robot/test/integration_test

```diff
@@ -84,18 +87,19 @@ class IntegrationTest:
 
     def check_result(self, result):
         if result == self.expected_result:
-            rospy.loginfo("test passed")
-            return
-        rospy.loginfo("test failed")
-        rospy.loginfo("result: {}".format(result))
-        rospy.loginfo("expected result: {}".format(self.expected_result))
+            return True
+        else:
```

Comment:
Don't need an else here as it's already returning if it's successful.

Reply:
Don't need it but it's more readable :)